docs/ioc/tutorials/streamdevice.rst

```diff
@@ -147,11 +147,11 @@ and what to expect in return.
    Terminator = LF;
 
    getVoltage {
-       out ":VOLT?"; in "%f";
+       out ":volt?"; in "%f";
```

Comment:
I'm surprised that it is now lower case. Usually SCPI commands are upper case.
Typically it's more :VOLTage or :VOLT? abbreviated
https://en.wikipedia.org/wiki/Standard_Commands_for_Programmable_Instruments

Reply:
Yeah, that's something I might need to change in the simulator. In another PR?

Reply:
On the SCPI-based power-supply that I controlled a while ago, it actually accepted both lower and uppercase. Documentation only mentioned uppercase, though. But I didn't find how to do it with Lewis.

Reply:
yeah sure